raft/raft.go

```diff
@@ -864,14 +870,14 @@ func (r *raft) loadState(state pb.HardState) {
 }
 
 // isElectionTimeout returns true if r.electionElapsed is greater than the
-// randomized election timeout in (electiontimeout, 2 * electiontimeout - 1).
+// randomized election timeout in [electiontimeout, 2 * electiontimeout - 1).
 // Otherwise, it returns false.
 func (r *raft) isElectionTimeout() bool {
-	d := r.electionElapsed - r.electionTimeout
-	if d < 0 {
-		return false
-	}
-	return d > r.rand.Int()%r.electionTimeout
+	return r.electionElapsed > r.randomizedElectionTimeout
```

Comment:
this changed the previous checking logic. But I feel changing this to >= is better.